mail-filter/dcc/dcc-1.3.158-r1.ebuild

```diff
@@ -0,0 +1,145 @@
+	)
+	milter? (
+		|| (
+			mail-filter/libmilter
+			mail-mta/sendmail
+		)
+	)
+	rrdtool? ( net-analyzer/rrdtool )"
+DEPEND="${RDEPEND}"
+
+dcc_cgibin=var/www/localhost/cgi-bin/dcc
+dcc_homedir=var/dcc
+dcc_libexec=usr/sbin
+dcc_man=usr/share/man
+dcc_rundir=var/run/dcc
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.3.140-freebsd.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
+
+src_configure() {
+	tc-export CC AR RANLIB
+	local myconf=(
+		--bindir="${EPREFIX}"/usr/bin
+		--homedir="${EPREFIX}"/${dcc_homedir}
+		--libexecdir="${EPREFIX}"/${dcc_libexec}
+		--mandir="${EPREFIX}"/usr/share/man
+		--enable-dccifd
+		--enable-server
+		--with-DDC-MD5
+		--with-installroot=${ED}
+		--with-rundir="${EPREFIX}"/${dcc_rundir}
+		--with-uid=root
+		--with-updatedcc_pfile="${EPREFIX}"/${dcc_homedir}/updatecc.pfile
+		--with-db-memory=64
+		--with-max-db-mem=128
+		--with-max-log-size=0
+		--with-make-cmd=${MAKE:-make}
+		$(use_enable ipv6 IPv6)
+		$(use_enable milter dccm)
+		$(use_with cgi cgibin ${dcc_cgibin})
+		$(use_with milter sendmail)
+	)
+
+	einfo "Using config: ${myconf[@]}"
+
+	# This is NOT a normal configure script.
+	./configure "${myconf[@]}" || die "configure failed!"
+}
+
+moveconf() {
+	local into=/etc/dcc/
+	for i in $@; do
+		mv "${ED}/${dcc_homedir}/${i}" "${ED}/${into}" || die
+		dosym ../../etc/dcc/"${i}" "${dcc_homedir}/${i}"
+	done
+}
+
+src_install() {
+	# stolen from the RPM .spec and modified for gentoo
+	export MANOWN=root
+	export MANGRP=$(id -g -n root)
+	export BINOWN="${MANOWN}"
+	export BINGRP="${MANGRP}"
+	export DCC_PROTO_HOMEDIR="${ED}/${dcc_homedir}"
+	export DCC_CGIBINDIR="${ED}/${dcc_cgibin}"
+	export DCC_SUID="${BINOWN}"
+	export DCC_OWN="${BINOWN}"
+	export DCC_GRP="${BINGRP}"
+
+	dodir /etc/cron.daily "${dcc_homedir}" /usr/bin /usr/sbin /usr/share/man/man{0,8} /etc/dcc
+	if use cgi ; then
+		dodir "${dcc_cgibin}"
+	fi
+	keepdir /var/log/dcc
+
+	emake install
+
+	# branding and setting reasonable defaults
+	sed -e "s/BRAND=\$/BRAND='Gentoo ${PF}'/;" \
+		-e "s/GREY_ENABLE=\$/GREY_ENABLE=off/;" \
+		-e "s/DCCM_LOG_AT=5\$/DCCM_LOG_AT=50/;" \
+		-e "s,DCCM_LOGDIR=\"log\"\$,DCCM_LOGDIR=\"/var/log/dcc\",;" \
+		-e "s/DCCM_ARGS=\$/DCCM_ARGS='-SHELO -Smail_host -SSender -SList-ID'/;" \
+		-e "s/DCCIFD_ARGS=\$/DCCIFD_ARGS=\"\$DCCM_ARGS\"/;" \
+		-e 's/DCCIFD_ENABLE=off/DCCIFD_ENABLE=on/' \
+		-e 's/DBCLEAN_LOGDAYS=14/DBCLEAN_LOGDAYS=1/' \
+		-i "${ED}/${dcc_homedir}/dcc_conf"
+
+	if use milter ; then
+		# enable milter
+		sed -i -e "s:^[\t #]*\(DCCM_ENABLE[\t ]*=[\t ]*\).*:\1on:g" \
+			"${D}/${dcc_homedir}"/dcc_conf
+	fi
+
+	# provide cronjob
+	mv "${ED}"/usr/sbin/cron-dccd "${ED}"/etc/cron.daily/dccd || die "mv failed"
+
+	# clean up
+	mv "${ED}"/usr/sbin/logger "${ED}"/usr/sbin/logger-dcc || die "mv failed"
+
+	statslist="${ED}/usr/sbin/{dcc-stats-graph,dcc-stats-init,dcc-stats-collect}"
+	if ! use rrdtool; then
+		# remove rrdtool interface scripts
+		eval rm -f ${statslist} || die "Failed to clean up rrdtool scripts"
```

Comment:
remove the `eval` and just inline the variable

Reply:
done